src/pages/Login/Login.tsx

```diff
@@ -9,30 +9,56 @@ import {useAppDispatch, useForm} from "../../utils/hooks";
 function Login() {
   const [value, handleChange] = useForm({"E-mail": "", "Password": ""});
   const [type, setType] = React.useState<"password" | "text" | "email">('password');
+  const [error, setError] = React.useState(false);
```

Comment:
I'd suggest more clear naming to improve readability. `type` and `error` are too generic.

```tsx
// also, remove "email" from possible values
const [passwordInputType, setPasswordInputType] = React.useState<"password" | "text">("password");
const [submitError, setSubmitError] = React.useState(false);
```